architecture/rhacs-cloud-service-service-description.adoc

```diff
@@ -0,0 +1,179 @@
+Red Hat does not provide user access to component logs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
+
+
+[id="scalability-and-service-level_{context}"]
+== Scalability and Service Level
+
+{rh-rhacscs-first} has set limits on the number of cores or clusters it can protect.
+The limits are based on the resources available in the Secured Cluster and the limits on usability.
+
+[id="service-level-objective-and-agreement_{context}"]
+=== Service Level Objective and Agreement
+
+For more information about Service Level Objectives (SLOs) and Service Level Agreements (SLAs) in link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
+
+
+[id="updates-and-upgrades_{context}"]
+== Updates and Upgrades
+
+Red Hat makes a commercially reasonable effort to notify customers prior to updates and upgrades that impact service.
+The decision regarding the need for a Service update to the Central instance and its timing is the sole responsibility of Red Hat.
+
+Customers have no control over when a Central service update occurs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
+Upgrades to the version of {rh-rhacscs-first} are considered part of the service update.
+
+The version of {product-title-managed-short} used on Secured Clusters must match the version of the Central instance of {product-title-managed-short} to ensure compatibility.
+Customers are responsible for Secured Cluster services upgrades required to maintain this version compatibility.
+
+[id="availability_{context}"]
+== Availability
+
+Availability and disaster avoidance are extremely important aspects of any security platform. {rh-rhacscs-first} provides numerous protections against failures at multiple levels.
+To account for possible cloud provider failures, Red Hat established multiple availability zones.
+
+
+[id="backup-and-disaster-recovery_{context}"]
+=== Backup and Disaster Recovery
+
+All {product-title-managed-short} clusters are backed up using Database backups.
+This also applies to customer data stored in the Central database.
+
+All snapshots are created using the appropriate cloud provider snapshot APIs, encrypted  and then uploaded to secure object storage, which for  Amazon Web Services (AWS) is an S3 bucket.
+
+* Red Hat does not commit to a Recovery Point Objective (RPO) or Recovery Time Objective (RTO). For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
+
+* Site Reliability Engineering (SRE) performs backups only as a precautionary measure. They are stored in the same region as the cluster.
```

Comment:
later "SRE" is identified as site reliability engineer(s) so maybe remove "(SRE)" here